flink-runtime/src/test/java/org/apache/flink/runtime/state/memory/ByteStreamStateHandleTest.java

```diff
@@ -172,4 +172,15 @@ public void testBulkReadINdexOutOfBounds() throws IOException {
             // expected
         }
     }
+
+    @Test
+    public void testStreamWithEmptyByteArray() throws IOException {
+        final byte[] data = new byte[0];
+        final ByteStreamStateHandle handle = new ByteStreamStateHandle("name", data);
+
+        FSDataInputStream in = handle.openInputStream();
+        in.seek(0);
+        byte[] dataGot = new byte[1];
+        assertEquals(0, in.read(dataGot, 0, 0));
```

Comment:
Do we need to test `assertEquals(0, in.read());`?

Reply:
In this context, in.read() should return -1, bacause read always expect one more byte.

Reply:
> Do we need to test `assertEquals(0, in.read());`?

In this context, in.read() should return -1, bacause read always expect one more byte.